types/types.go

```diff
@@ -4,4 +4,5 @@ package types
 
 const (
 	PubSubChangesTopic = "changes"
+	UibffEndpoint      = "https://api.dev.streamdal.com"
```

Comment:
We should change this right now so we don't forget to do it later.

Reply:
Done!